web/modules/custom/joinup_news/joinup_news.module

```diff
@@ -53,14 +52,7 @@ function joinup_news_workflow_selector(EntityInterface $entity) {
   }
   /** @var \Drupal\joinup_core\JoinupRelationManager $relation_manager */
   $relation_manager = \Drupal::service('joinup_core.relations_manager');
-  try {
-    $moderation = $relation_manager->getParentModeration($entity);
-  }
-  catch (MissingRelationException $e) {
-    // Throw a more helpful exception if the news article doesn't have a parent
-    // collection or solution.
-    throw new \Exception('Cannot determine the moderation workflow type on a news article that doesn\'t belong to a collection or solution.');
-  }
+  $moderation = $relation_manager->getParentModeration($entity);
```

Comment:
Oh I see.. it reaches up to MembershipManager::getGroups where it tries to access a method.. Cool. I will revert this one..